tests/unit/AGMDate.spec.ts

```diff
@@ -13,7 +13,7 @@ Vue.use(Vuetify)
 Vue.use(Vuelidate)
 
 const vuetify = new Vuetify({})
-const store = getVuexStore()
+const store = getVuexStore() as any // remove typings for unit tests
```

Comment:
This is needed because we don't want to use types in the unit tests.

For example, we want to use "CP" instead of "EntityType.COOP" in the tests, so that we catch changes to enums. So the store is now untyped (in the unit tests only).